packages/site/src/components/BodyBlock.tsx

```diff
@@ -1,32 +1,21 @@
-import { Box, Content, Typography } from "@jobber/components";
+import { Content } from "@jobber/components";
 import { PropsWithChildren } from "react";
 
-interface BodyBlockProps extends PropsWithChildren {
-  readonly title: string;
-}
-
 /**
  * A simple opinionated wrapper for a block of content within a PageBlock
- * @param param0 {children:ReactNode, title: string}
+ * @param param0 {children:ReactNode}
  * @returns ReactNode
  */
-export const BodyBlock = ({ children, title }: BodyBlockProps) => {
+export const BodyBlock = ({ children }: PropsWithChildren) => {
```

Comment:
I _think_ removing the title here is what's making the site build CI step fail

Reply:
Ohhhhh yup, had to remove the body title from a couple of other places as well